PROJECT_LIFECYCLE.md

```diff
@@ -155,6 +155,31 @@ You should use the [![Archived badge](https://img.shields.io/badge/Stage-Archive
 [![Archived badge](https://img.shields.io/badge/Stage-Archived-orange)](https://github.com/eiffel-community/community/blob/master/PROJECT_LIFECYCLE.md#stage-archived)
 ```
 
+#### Process for archiving a repository
+
+- Create ticket in the repository to be archived. See [This repo should be archived eiffel-store#10](https://github.com/eiffel-community/eiffel-store/issues/10) for an example
+- Send mail to maintainers google group, with the following contents (as decided in TC on [Feb 27](https://github.com/eiffel-community/community/blob/master/meetings/MEETINGS_TC_2024.md#action-items-7)):
```

Comment:
That URL is outdated and doesn't point to the Feb 27 meeting. Also, since PROJECT_LIFECYCLE.md and these instructions are intended for the long term we should probably be more specific than just "Feb 27". Maybe state the full date of the TC meeting and link to MEETINGS_TC_2024.md?